config/oidc.php

```diff
@@ -31,5 +31,6 @@
 
     'callback_route_path' => env('OIDC_CALLBACK_ROUTE_PATH', 'callback'),
     'redirect_path_after_login' => env('OIDC_REDIRECT_PATH_AFTER_LOGIN', '/'),
-    'redirect_path_after_logout' => env('OIDC_REDIRECT_PATH_AFTER_LOGOUT', '/')
+    'redirect_path_after_logout' => env('OIDC_REDIRECT_PATH_AFTER_LOGOUT', '/'),
+    'disable_oidcstatemiddleware_for_post_callback' => false,
```

Comment:
You should add an env variable since this is a boolean

Reply:
Hi @maicol07,
I let you review the last commit.
Thanks